src/api/app/models/token.rb

```diff
@@ -1,10 +1,26 @@
 class Token < ApplicationRecord
-  belongs_to :user, foreign_key: 'user_id', inverse_of: :service_tokens
+  belongs_to :user
   belongs_to :package, inverse_of: :tokens
 
   has_secure_token :string
 
   validates :user, presence: true
+
+  def token_name
+    self.class.token_name
+  end
+
+  def self.token_type(action)
```

Comment:
this can be returned directly in the methods.

Reply:
@Ana06 could be more specific, please?